doc/source/index.rst

```diff
@@ -78,9 +97,37 @@ cloud-based deployments and tips for maintaining your deployment.
    security
    upgrading
    cost
-   faq
+   troubleshooting
    advanced
 
+.. _community-resources:
+
+Resources from the community
+----------------------------
+
+This section gives the community a space to provide information on setting
+up, managing, and maintaining JupyterHub.
+
+.. important::
+   We recognize that Kubernetes has many deployment options. As a project team
+   with limited resources to provide end user support, we rely on community
+   members to share their collective Kubernetes knowledge and JupyterHub
+   experiences.
+
+We hope that you will use this section to share deployments with on a variety
+of infrastructure and for different use cases.
```

Comment:
does "variety of infrastructure" parse? maybe it should be "variety of deployments"?

Reply:
Yeah. I see a typo too. Please open a PR to reword.